lib/src/pdf/pdf-worker.ts

```diff
@@ -1,16 +1,29 @@
-import rustPlugin from '../../pkg/obsidian_text_extract_bg.wasm'
-import * as plugin from '../../pkg'
+import PDFJSStatic from 'pdfjs-dist';
+//import getDocument from 'pdfjs-dist';
 
-const decodedPlugin = decodeBase64(rustPlugin as any)
+//import * as pdfjslib from 'pdfjs-dist';
+//let PDFPageProxy = pdfjslib.PDFJS;
 
-onmessage = async evt => {
-  const buffer = Uint8Array.from(decodedPlugin, c => c.charCodeAt(0))
-  await plugin.default(Promise.resolve(buffer))
+onmessage = async path => {
```

Comment:
Log it to make sure, but this should probably stay `async evt =>`. This call is triggered when you call `worker.run()` in pdf-manager.ts, so you should receive an event object of the shape
```
{
  data: {
    path: string,
    name: string
  }
}
```

Reply:
updated